src/ontology/cl-edit.owl

```diff
@@ -25654,13 +25655,14 @@ SubClassOf(obo:CL_1000346 obo:CL_0002071)
 
 # Class: obo:CL_1000347 (colonocyte)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "PMID:34497389") obo:IAO_0000115 obo:CL_1000347 "A columnar cell of the colon that is part of the colonic epithelium.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "PMID:34497389") Annotation(oboInOwl:hasDbXref "PMID:16870803") Annotation(oboInOwl:hasDbXref "PMID:23821742") obo:IAO_0000115 obo:CL_1000347 "A columnar cell that is part of the colonic epithelium. This cell is responsible for the absorption, transport, and metabolization of short-chain fatty acids (SCFAs) produced by gut bacteria, as well as the transport and absorption of water and electrolytes.")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_1000347 "FMA:263114")
```

Comment:
We have enterocyte as a parent cell type so I would mention that in the first sentence.

Maybe something like:

An enterocyte (absorptive epithelial cell) of the colonic epithelium.